### PR TITLE
Propose structured agenda for working group

### PR DESCRIPTION
**Pull request by natalieparellano**
From previous conversations, here are some ideas to improve our working group:

We should:
* time box topics - sometimes one topic can take up most of the meeting. If we find ourselves exceeding 15 minutes on a topic, we should move on. We could continue the discussion on the RFC PR, schedule a separate meeting, or open a discussion topic in buildpacks/community.
* announce topics ahead of time - this gives people a chance to review, think, and comment on RFCs and also to know if the content of the meeting will be interesting to them when deciding whether to join. The downside is that sometimes organic discussions need to happen. We could leave space for this at the beginning or end of the meeting.
* (maybe) schedule topics for a specific time - this would allow people to join only for the topics that interest them. The downside is that if we cover something in a shorter time than scheduled it will mess up the schedule or we’ll be waiting around…

With all that said, this PR proposes a structured agenda for working group. We could use any free time that opens up from discussions that end early to cover “quick” topics or silent read the RFC for the next topic.

### CODE REVIEW

File: README.md
Comment (aemengo) [inline README.md:52]: Perhaps working group isn’t the correct forum, but I’d benefit from a segment explicitly about **Priority RFCs/Issues**.

There’s loads to work on, but our community isn’t large enough where all initiatives can be executed at the same time. And not all issues have the same level of impact. As a contributor, I’d like to know which initiatives (_and what topics_) have the highest impact. What _I_ want to work on isn't necessarily what the steering committee has agreed upon.

My best answer for this is our [ROADMAP](https://github.com/buildpacks/community/blob/main/ROADMAP.md) (_which is great_) but I only see 2 RFCs listed there. Participate in the community enough you can get a sense of "what’s not being listed", but writing the RFC requires a lot of context and time commitment (_it's certainly difficult for me_). This segment could be a middle ground.

Comment (hone) [reply README.md:52]: @aemengo this is great feedback. Would you want this on a weekly cadence? Monthly? Quarterly?

Comment (aemengo) [reply README.md:52]: Monthly feels appropriate.

File: README.md
Comment (hone) [inline README.md:57]: Is it someone's job to decide what the topics are if there are more than 3? Is it first come first serve?

Comment (natalieparellano) [reply README.md:57]: I think in practice it might be first come first serve, but we could leave it up to the discretion of the organizers.

Looking at https://github.com/buildpacks/community/pull/107, do you think review of open RFCs will move to that meeting? That might free up some time in WG to cover more topics or just leave space for ad-hoc discussion.

Comment (hone) [reply README.md:57]: yeah, that's my intent but I'm open to feedback.

> I think in practice it might be first come first serve, but we could leave it up to the discretion of the organizers.
It's probably worth being clear on how that's done.

Stuff gets added to Future Topics and the organizer picks 3 topics the day before.

> Looking at #107, do you think review of open RFCs will move to that meeting? That might free up some time in WG to cover more topics or just leave space for ad-hoc discussion.
That's the plan, but I'm open to feedback.

**Review by yaelharel — COMMENTED** [review]
I really like this PR!!! I think that it’ll make our working groups much more efficient and it’ll be easier for new contributors to follow it.
I think that time box topics and announce topics ahead of time is great.
I’m a bit worried about scheduling topics for specific time for the same reason that you mentioned in the first comment of this PR - if we cover something in a shorter time (one of the topics or the first part of  the meeting with “introductions, release updates, housekeeping”), it’ll mess up the schedule or we’ll be waiting.
I don’t think we should use any free time that opens up from discussions that end early for silent read because usually all of the maintainers and core team members join this meeting so it’s time that we can invest in other things. Also I don’t think that using this time to cover “quick” topics will work, because if you end a topic 2-3 minutes early, usually it won’t be enough time to cover even a “quick” topic.
I think it’s better to have more than 3 topics on the list. We should still block each one of them with 15 minutes, but in case we end talking about one of the topics early, we should move on to the next topic on the list and try to cover as much as we can. I understand that in this way people won’t be able to join part of the meeting, but I have a feeling that most people will either join the whole meeting or part of it based on their schedule and not based on its topics. Moreover, since it’s a meeting that happens only once a week, I think it’s better to try to cover as much as we can over allowing people to join only part of it.

File: README.md
Comment (natalieparellano) [reply README.md:57]:
```suggestion
Topics should be added to the agenda in the public [Google Doc](https://docs.google.com/document/d/1CwixRknj3gAXbi-fTI1-chwtMfy6G3GhdH17aEUjhK0/edit) the day prior to the meeting. The final list of topics will be finalized by the meeting organizers. If any scheduled topics are covered in less than 15 minutes, additional topics may be covered. 
```

**Comment by natalieparellano** [comment]
> I think it’s better to try to cover as much as we can over allowing people to join only part of it.

@yaelharel this makes sense to me. Added a suggestion: https://github.com/buildpacks/community/pull/106/files#r661734919

File: README.md
Comment (yaelharel) [reply README.md:57]: @natalieparellano, I like your suggestion but I think we should also remove the times on lines 52-55, otherwise it can be confusing when we're going to cover the additional topics (between the 3 main topics or at the end).

Maybe, a day before the meeting:
- Topics should be added to "future topics" by the morning (for example 9am EST).
- Organizers should create 3 lists for the meeting by noon (EST):
  1. introductions, release updates, housekeeping (up to 15 minutes in total)
  2. main topics (up to 15 minutes each)
  3. additional topics (if we have time)

In this way, community members will have ~0.5 day to review the agenda before the meeting.

**Comment by jabrown85** [comment]
I love this! The additional structure with time limits is the my favorite bit - I think one thing that might be missing is what happens when someone runs out of time for the topic. I think a more formal - `continue this discussion async in #slack-channel` mechanism may be needed. Maybe having a `#working-group` channel so folks can dump questions during a talk without interrupting would help drive more async communication.

File: README.md
Comment (natalieparellano) [reply README.md:52]: IIUC "roadmap review" will be a quarterly recurring item for the Core Team Sub Sync (on Wednesdays).

**Comment by samj1912** [comment]
Should we merge this given that we have roughly been following this for the last meetings?

**Comment by natalieparellano** [comment]
@samj1912 sounds good! The conflicts should be fixed now.

**Review by samj1912 — APPROVED** [review]

**Review by jromero — APPROVED** [review]

**Pull request merged**
